src/snapshotVersion.js

```diff
@@ -3,9 +3,19 @@ const createVersion = require('./lib/createVersion.js');
 
 module.exports = (argsParser = process.argv) => {
     program
-        .command('create <version> [siteDir]')
-        .action((version, siteDir = ".") => {
-            createVersion.create(version, siteDir);
+        .command('create')
+        .storeOptionsAsProperties()
```

Comment:
If we upgrade `commander` to latest stable release, we probably don't need to use it anymore. Besides, this PR is changing the way CLI parameters are being processed. So, it will have to be a `major` release anyways. Should we just upgrade `commander` as part of this PR?

Reply:
Yeah, we might as well upgrade it.